habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/exec/PythonVersionHelper.java

```diff
@@ -21,17 +22,17 @@ public class PythonVersionHelper {
 
     private static final String PYTHON_COMMAND = "python";
     private static final String PYTHON_3_COMMAND = "python3";
-    private static final String pythonVersion3Regex = "^3.*";
-    private static final String extractVersionRegex = "^.*?(?=([0-9]))";
+    private static final String PYTHON_VERSION_3_REGEX = "^3.*";
+    private static final String EXTRACT_VERSION_REGEX = "^.*?(?=([\\d]))";
```

Comment:
A: Does it need to be in a character class block if we're using the built-in character class?
```suggestion
    private static final String EXTRACT_VERSION_REGEX = "^.*?(?=(\\d))";
```

Reply:
Good catch - updated ✅